src/test/compile-fail/issue-28433.rs

```diff
@@ -8,16 +8,12 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-use zoo::bird::{duck, goose};
-
-mod zoo {
-    pub enum bird {
-        pub duck, //~ ERROR: unnecessary `pub` visibility
-        goose
-    }
+enum bird {
+    pub duck, //~ ERROR: expected identifier, found keyword `pub`
```

Comment:
I’d suggest waiting for travis build to finish for now.

Otherwise the fastest way without editing makefiles is to do `make -j8 check-stage2-cfail TESTNAME=issue-28433` I think.